python/ray/experimental/sgd/pytorch/distributed_pytorch_runner.py

```diff
@@ -98,8 +105,8 @@ def step(self):
             self.train_loader.sampler.set_epoch(self.epoch)
         return super(DistributedPyTorchRunner, self).step()
 
-    def get_state(self):
-        """Returns the state of the runner."""
+    def _get_model_state_dicts(self):
```

Comment:
Aren't `_get_mode_state_dicts` and `_set_model_state_dicts` identical to the parent class implementation now?

Reply:
Added a better docstring

Reply:
"Fetch state from ``model.module``, which is needed for PyTorch DDP."